x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/dataframe/extractor/ExtractedFieldsDetector.java

```diff
@@ -58,9 +59,7 @@
     /**
      * Fields to ignore. These are mostly internal meta fields.
      */
```

Comment:
Comment looks to be out of date now it's a list of one.

Reply:
Thanks, I pushed https://github.com/elastic/elasticsearch/pull/69977/commits/78fb83ed12c190e52aa24eca0cd8b814271f5bee